flambe/field/label.py

```diff
@@ -27,21 +27,29 @@ def __init__(self,
         multilabel_sep : str, optional
             If given, splits the input label into multiple labels
             using the given separator, defaults to None.
-        labels: Sequence[str], optional
+        labels: Union[str, List[str]], optional
             If given, sets the labels and the ordering is used to map
             the labels to indices. That means the first item in this
             list will have label id 0, the next one id 1, etc..
             When not provided, indices are assigned as labels are
-            encountered during preprocessing.
+            encountered during preprocessing. The list can also be
+            provided as a file with a label on each line.
 
         """
         self.one_hot = one_hot
         self.multilabel_sep = multilabel_sep
         self.tokenizer = LabelTokenizer(multilabel_sep=self.multilabel_sep)
 
         if labels is not None:
+            if isinstance(labels, str):
```

Comment:
I'd use a registered factory to avoid coupling the implementation to a specific format/way of building Also, it will allow easier testing, since you could test separately the way its built from the internal operations

Reply:
I agree in spirit, but do you think it's very valuable here? I feel like the factory method will be so trivial it might not be worth it

Reply:
I hesitated a bit before going for this implementation, but I'm ok with going the classmethod route

Reply:
Well, apart of helping the developers in the future (they could add new ways of building the thing without needing to change the code), it could help users right?  The code would be more explicit on the fact that a path for a file is being used

Reply:
yeah i ll change it!